external-crates/move/crates/move-package-alt/src/dependency/dependency_set.rs

```diff
@@ -9,9 +12,16 @@ use crate::package::{EnvironmentName, PackageName};
 /// A set of default dependencies and dep overrides. Within each environment, package names are
 /// unique.
 ///
-/// Iterating over a dependency set produces (Option<EnvironmentName>, PackageName, T) tuples; the
-/// environment name is None to represent the default environment. See [DependencySet::iter] for
-/// more details.
+/// Iterating over a dependency set produces Option<[EnvironmentName]>, [PackageName], T) tuples for
+/// all declared dependencies (an environment name of `None` represents the default environment).
+///
+/// Note that this doesn't do any merging or inheritance - default dependencies are returned with
```

Comment:
Check if this comment was fixed
```suggestion
/// Note that iteration (and most operations) does not do any merging or inheritance - default dependencies are returned with
```